README.md

```diff
@@ -4,7 +4,27 @@ TypeScript based types used in PNC project.
 
 This project clones **PNC project** and builds `pnc/rest-api` module to get `pnc-openapi.json` specification using `generate-openapi-spec.sh` script and transform it to `index.d.ts` using `dtsgen` command.
 
-## Generate new types and release new version
+## Releasing new version of pnc-api-types-ts
+
+### Remote & automatic
+
+#### Option A - fully automatic
+
+New pnc-api-types-ts release is created by our jenkins instance automatically when there is new tag created in **PNC** project repository.
+
+#### Option B - one user click automatic
+
+When you need to create new pnc-api-types-ts release from unreleased PNC revision you can go to our jenkins instance and run `pnc-api-types-release` job. Job will ask for **PNC** revision to create new release from and takes care of rest.
+
+### Local
```

Comment:
I would mark it as a deprecated way `### Local (deprecated, only if Remote is not working)`